server/test/forms/translation/MultiOptionQuestionTranslationFormTest.java

```diff
@@ -36,7 +37,7 @@ public void buildsQuestion_newLocale_savesUpdates() throws Exception {
         (MultiOptionQuestionDefinition) form.builderWithUpdates(question, Locale.CHINA).build();
 
     assertThat(updated.getOptionsForLocale(Locale.CHINA))
-        .containsExactly(LocalizedQuestionOption.create(1L, 1L, "new", Locale.CHINA));
+        .containsExactly(LocalizedQuestionOption.create(1L, 1L, "opt1", "new", Locale.CHINA));
```

Comment:
This file is different then the others in the PR, but it's the only other place `LocalizedQuestionOption.create()` is used so I updated it as well.